.travis.yml

```diff
@@ -5,4 +5,6 @@ cache:
   directories:
     - node_modules
 script:
-  - npm test
\ No newline at end of file
+  - npm test
+  - npm run prettier
```

Comment:
what for do we need prettier here? results won't be saved in repo

I was talking about using `precommit` hook for formatting code

Reply:
Oh got it. I'll change it to a precommit hook that will change the formatting before committing to the repo.